.github/workflows/ci.yml

```diff
@@ -246,7 +246,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        mapdl-version: ['v21.1.1', 'v21.2.1', 'v22.1.0']
+        mapdl-version: ['v21.1.1', 'v21.2.1', 'v22.1.0', 'v22.2.0']
```

Comment:
Did you want to drop `v21.1.1` now too?

Reply:
Is v21.1

> > @germa89, is the database feature broken on v22.2?
> 
> Yes. That's why nightly MAPDL fails. @dts12263 is aware and @FredAns needs to implement some changes.

If this fails we need to disable this feature for v22.2. If anyone tries to use this it's going to fail outright.

Reply:
Done.